libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -314,7 +314,7 @@ - (void)testCreateQuerySearchDelete {
 // - query updated object (requestForSearch) and make sure we don't get anything
 - (void)testCreateUpdateQuerySearchDelete {
     // create object
-    NSString *lastName = [NSString stringWithFormat:@"Doe-%@", [NSDate date]];
```

Comment:
Good catch @khawkins! @huminzhi We should probably close this PR and reopen the bug you filed :-)